packages/backend/src/history/historyNotifier.ts

```diff
@@ -39,7 +39,7 @@ export class HistoryNotifier implements Disposable {
     await this.webview.postMessage({
       id: Messages.MSG_HISTORY_UPDATE,
       // Must pass in an empty body to satisfy the type system, if it is undefined, this fails.
-      body: '',
+      body: {},
```

Comment:
can be removed (no body parameter)

Reply:
yes, exact, reading I thought we had access to the notify method (and calling that notify method)

Reply:
There are unfortunately two places bootc backend calls webview.postMessage(), both need to pass a body and I was just trying to fix both calls here. IMHO this instance should call api-imp.ts notify instead of webview directly, but wasn't trying to fix that tech debt at the same time.

Reply:
sure, it was me that read incorrectly